packages/server/src/supertokens.ts

```diff
@@ -222,22 +243,22 @@ export class SessionContextClass implements SessionContext {
     return this._kernel.publicData
   }
 
-  $authorize(input?: any) {
+  $authorize(...args: Parameters<Authorize>) {
```

Comment:
This usage of `Authorize` shouldn't be exposed to the end user app. The type definition for this class comes from `@blitzjs/core` which is where users import it from. This here in `@blitzjs/server` is only the implementation.